### PR TITLE
Mp/multihost host headers

### PR DESCRIPTION
**Pull request by marcparadise**
Do not 500 if multiple hosts present in X-Forwarded-Host/Server

X-Forwarded-For-Host/Server can contain a comma-separated list of hosts.

This change recognizes that and extracts the first host from such a list
if it's used.

Reference: https://httpd.apache.org/docs/current/mod/mod_proxy.html#x-headers.

### CODE REVIEW

File: test/decision_core_test.erl [hunk 1]
```diff
@@ -252,26 +252,6 @@ core_tests() ->
      fun head_method_not_allowed/0,
      fun non_standard_method_501/0,
      fun non_standard_method_200/0,
-     fun bad_request_b9/0,
```
Comment (seancribbs) [inline test/decision_core_test.erl:255]: Why were these tests removed?

Comment (marcparadise) [reply test/decision_core_test.erl:255]: Nice catch, they shouldn't have been.  I was narrowing down a timeout in one of the tests, looks like I messed up adding it back  - fixing now.

**Comment by marcparadise** [comment]
I've updated for the above, as well as fixing the other  inadvertent change (using HTTP 1.0 in the  `nonstandard_method_501` test instead of  HTTP 1.1)

**Comment by seancribbs** [comment]
👍

**Pull request merged**